doc/man1/req.pod

```diff
@@ -255,6 +256,15 @@ be a positive integer. The default is 30 days.
 Serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
 
+=item B<-extension ext>
+
+Add a specific extension to the certificate (if the B<-x509> option is
+present) or certificate request.  The argument must have the form of
+config key/value pairs (i.e. the lines you commonly find in the config
+file).
+
+This option can be given multiple times, it's values will be accumulated.
```

Comment:
"their values", no?

Reply:
No, as it's *one* option.  But "it's" should be "its"...  will fix

Reply:
Yes, but it's "multiple times". Indeed, one can as well say "option can be given multiple times, *all* specified key-value pairs will be accumulated."